src/pgduckdb_node.cpp

```diff
@@ -77,16 +77,16 @@ ExecuteQuery(DuckdbScanState *state) {
 			auto &executor = duckdb::Executor::Get(*connection->context);
 			// Wait for all tasks to terminate
 			executor.CancelTasks();
-
 			// Delete the scan state
 			CleanupDuckdbScanState(state);
 			// Process the interrupt on the Postgres side
 			ProcessInterrupts();
-			elog(ERROR, "Query cancelled");
+			elog(ERROR, "(PGDuckDB/ExecuteQuery) Query cancelled");
```

Comment:
I don't think the prefix really adds anything useful here.
```suggestion
			elog(ERROR, "Query cancelled");
```

Side-note: We should be throwing the expected error code here.